orchestra/actions/clone.py

```diff
@@ -86,18 +85,20 @@ def branch(self):
         return None, None
 
     def _branches_from_remote(self, remote):
-        env = dict(self.environment)
-        env["GIT_SSH_COMMAND"] = "ssh -oControlPath=~/.ssh/ssh-mux-%r@%h:%p -oControlMaster=auto -o ControlPersist=10"
+        try:
+            result = quiet_get_script_output(f'git ls-remote -h --refs "{remote}"', environment=self.environment)
 
```

Comment:
How is this better than
```python
result = try_get_script_output(...)
if result is None:
  return {}
...

Reply:
The only other action where a helper is used directly is in `install.py:_build_and_install`, because it needs to pass the `RUN_TESTS` env var.
The value of the variable depends on `args` which we don't get, so we can't move it inside `InstallAction.environment` and use `Action._run_user_script`